Gemfile

```diff
@@ -1,4 +1,2 @@
 source 'https://rubygems.org'
 gemspec
-
-gem 'redis'
```

Comment:
Another way to convey the message is to use conditional inclusion in the Gemfile like:

```
gem 'redis' if ENV['REDIS']
```

This approach will still need the `NoRedisError` message, but the text can be updated to reflect the procedure or passing environment variable while running `bundle install`. This will have an added advantage of tracking the desired version number of the gem in the Gemfile. Additionally, anyone looking at the Gemfile will notice the need of the gem.